client/api/disk/v1beta3/api.proto

```diff
@@ -41,69 +41,71 @@ message DiskLocation {
 }
 
 message ListDiskLocationsResponse {
-    // Map of disk device IDs and <adapter, bus, target, lun ID> associated with each disk device
-    map <string, DiskLocation> disk_locations = 1;
+    // Map of disk number and <adapter, bus, target, lun ID> associated with each disk device.
+    map <uint32, DiskLocation> disk_locations = 1;
```

Comment:
could we define diskLocations key similar to diskIDs so that it is clear to see what are the keys in the map?

Reply:
I think this is the same as diskIDs

```
message DiskIDs {
    // The disk page83 id.
    string page83 = 1;
    // The disk serial number.
    string serial_number = 2;
}

message ListDiskIDsResponse {
    // Map of disk numbers and disk identifiers associated with each disk device.
    map <uint32, DiskIDs> diskIDs = 1;  // the case is intentional for protoc to generate the field as DiskIDs
}
```

Reply:
ack